assembly.xml

```diff
@@ -12,7 +12,7 @@
             <directory>src/main/resources/camel</directory>
             <outputDirectory>routes</outputDirectory>
             <includes>
-                <include>odoo-*.xml</include>
+                <include>**/odoo-*.xml</include>
```

Comment:
Now we have routes that don't have this **odoo** prefix, so you have just include all routes